notes/area.product.sop.ready-to-ship.md

```diff
@@ -16,8 +16,11 @@ How to add an entry to [[Ready to Ship|dendron://private/weekly.temp.team-journa
 1. After a PR has been merged, add it to the correct triage area in the weekly [^ship]
 ![[dendron://dendron.handbook/area.product.sop.ready-to-ship.notice-release-schedule]]
 1. Ensure that the correct type [^type] and scope [^scope] are assigned
-1. Add the alias of the alias of the user. 
+1. If the PR has multiple features, add in each feature separately. Feature breakdown should be included in the PR details when [[Submitting|dendron://dendron.docs/dev.process.review#submitting]] it. 
+1. Ensure the format for [[Pull Request title|dendron://dendron.docs/dev.process.review#pull-request-title]] is followed
+1. Add the alias of the alias of the user
```

Comment:
nitpick: `alias of the` is repeated

Reply:
updated, will also resolve the merge conflicts.

Reply:
was able to add directly to main through `f8542d3` with the changes, should I close this PR?

Reply:
You mean the whole PR? If so, sure, there's no point in merging the PR then.

Reply:
Yup, okay will close this then! Thanks!